app/src/main/AndroidManifest.xml

```diff
@@ -72,7 +72,7 @@
             android:configChanges="orientation|screenSize" />
         <activity android:name=".activities.IssueActivity" android:exported="false" />
         <activity android:name=".activities.IssueEditActivity" android:exported="false" />
-        <activity android:name=".activities.IssueLabelListActivity" android:exported="false" />
+        <activity android:name=".activities.issues_label.IssueLabelListActivity" android:exported="false" />
```

Comment:
Please don't mix in refactorings like those package moves. They make reviewing harder; and I don't really want this package move either.

Reply:
I'm sorry, you're right, I did not think of that ...

Reply:
Yes, please revert unrelated changes. They just make it harder to review.